### ISSUE TITLE
NVDA always announces 'blank' instead of the actual value for SpinButton with suffix.

### ISSUE THREAD

**Issue by VivianSi233** [microsoft/fluentui#22012]
### Environment Information

Fluent UI React 8.60.0
Browser: Edge 99.0.1150.30 (bit64)
OS: Win10

### Describe the issue:
1.Turn on NVDA(focus mode)
2.Go to https://developer.microsoft.com/en-us/fluentui#/controls/web/spinbutton
3.Focus on spinbutton with label above, press up/down arrow
4.Check actual result

#### Actual behavior:

NVDA always announces 'blank' instead of the actual value for SpinButton with suffix. 

#### Expected behavior:

NVDA should announce actual value for spinbutton with suffix.

**Comment by smhigley** [comment]
This is a known NVDA bug where it doesn't expose the value of inputs with `role="spinbutton"`: https://github.com/nvaccess/nvda/issues/13195

**Comment by VivianSi233** [comment]
Test Environment:
Test URL: [testcaseforadddatasource - Microsoft 365 compliance](https://df.compliance.microsoft.com/advancedediscovery/cases/v2/50fa33d4-b099-4b0c-a475-8e0c8ba4681a?casename=testcaseforadddatasource&casesworkbench=JobsTab&tid=596ade3a-1abe-4c5b-b7d5-a169c4b05d4a)
Test Account: [admin@ediscoSdf.onmicrosoft.com ](mailto:admin@ediscoSdf.onmicrosoft.com)
OS: Windows 11
Version: 21H2 (OS Build 22000.466)
Browser version: 99.0.1135.5 (Official build) dev (64-bit)
Screen Reader: NVDA

Repro step:
1. Open the above URL and login with valid credentials
2. Navigate to the 'Settings' tab and activate it.
3. Further select 'Search & analytics' select button in the displayed 'Search & analytics' pane navigate to spin button present in the pane and observe the NVDA announcement.

Actual Result:
1.When focus lands on 'Document and email similarity threshold ' spin button it's not announcing selected value, minimum and maximum value.
Ex: NVDA Announces 'Document and email similarity threshold spin button editable selected
2.Screen reader is not announcing the updated value in the spin button on pressing up and down arrow NVDA announce as blank.

Expected Result:
1.When focus lands on 'Document and email similarity threshold ' spin button Screen reader should announce selected value , minimum and maximum value.
Ex: NVDA should Announce 'Document and email similarity threshold spin button editable minimum 40 and maximum 100 selected 95'
2.Screen reader should announce the updated value in the spin button 
Ex: NVDA should Announce 'Document and email similarity threshold spin button editable selected 96'

Note:
Issue repro in entire Search & analytics pane.

Observation:
Issue doesn't repro with Narrator

User Impact:
Visually challenged users who rely on screen readers are impacted and will not be able to identify the correct information as Screen reader is not announcing the value in the spin button. This will definitely confuse the user, and this may lead to disorientation due to this issue.
